### DISCUSSION TITLE
Option to disable explosion of generated end crystals in the obsidian pillars

### DISCUSSION THREAD

**Discussion by lleeoo1999** [PurpurMC/Purpur#514] [__Completed__] [answered]
An option to disable generated end crystals in the obsidian pillars from destroying nearby blocks when summoning the enderdragon. This will allow building around the pillars without having to worry about it exploding.

**Comment by katekerllenevich** [comment, marked as answer]
You should be able to get this effect by changing `blocks.end-crystal.base.explosion-effect` to `NONE`

**Reply by NinjagoKristian** [reply]
What file is this in?

**Reply by encode42** [reply]
https://purpurmc.org/docs/Configuration/explosion-effect_3
